doc/classes/Node.xml

```diff
@@ -1068,6 +1068,11 @@
 				Emitted when the node's editor description field changed.
 			</description>
 		</signal>
+		<signal name="editor_state_changed">
+			<description>
+				Emitted when field relevant to the editor changed. Only emitted in the editor, to update the scene tree.
```

Comment:
Once this is finalized, and if this signal would still exist... this is not good. We need to be more specific. For now:
```suggestion
				Emitted when an attribute of the node that is relevant to the editor changed. Only emitted in the editor.
```

Reply:
I re-used the wording from : tree_process_mode_changed() `Emitted when the Node.process_mode of any node inside the tree is changed. Only emitted in the editor, to update the visibility of disabled nodes.`

Should we change that one too then?

Reply:
I don't see why the class reference would have to tell the reader this. Nowhere is "_to update the visibility of disabled nodes_" something they can reach through scripting, hence it's none of their business. So yeah?

Reply:
Alright, I won't update that in this PR, but either you and me and do a PR to fix up that little bit of doc then!